firebase-vscode/webviews/SidebarApp.tsx

```diff
@@ -141,6 +153,7 @@ export function SidebarApp() {
           }}
         />
       )}
+      {firebaseJson && <EmulatorPanel firebaseJson={firebaseJson} />}
```

Comment:
Probably gate this on `!!projectId` so that it's consistent with the deploy/hosting init panels? I guess you don't need to be logged in to use emulators, or else also `!!userEmail`.

Reply:
Really we only care about firebaseJson existence - we could combine that with userEmail if we want to only show the panel after the user has logged in, something like

`!!userEmail && !!firebaseJson && <...>`

WDYT?